k8s/zeppelin-server.yaml

```diff
@@ -217,10 +217,12 @@ kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: zeppelin-server-role-binding
+  namespace: default
 subjects:
 - kind: ServiceAccount
   name: zeppelin-server
+  namespace: default
```

Comment:
Please remove the namespace parameter. Afterwards, the RoleBinding is created in the namespace in which the other resources are also created.

Reply:
Sure.